file.go

```diff
@@ -1,37 +1,115 @@
-package jcsv
+package main
 
-import "os"
+import (
+	"encoding/csv"
+	"fmt"
+	"io/ioutil"
+	"os"
+)
 
 type file struct {
-	// TODO: define variables if needed
+	data []uint8
+	csvData [][]string
```

Comment:
Reading this code, I can't understand what is the difference between these two. There are no comments either.